openstack/orchestration/v1/stackresources/doc.go

```diff
@@ -2,4 +2,73 @@
 // A resource is a template artifact that represents some component of your
 // desired architecture (a Cloud Server, a group of scaled Cloud Servers, a load
 // balancer, some configuration management system, and so forth).
+
+/*
```

Comment:
doc nit: same fix as previously mentioned: let's change this all to use one big `/* */` comment.

Reply:
Done